resources/js/Components/Public/FullWidth/SummerCamps.vue

```diff
@@ -40,6 +40,12 @@
     <div class="flex flex-wrap items-center gap-4 lg:justify-center">
       <a :href="route('pirmakursiuStovyklos')">
         <NButton round size="large" strong type="primary"
+          >Sužinok apie savo artėjančią stovyklą!<template #icon
+            ><NIcon :component="Balloon20Regular"></NIcon></template
+        ></NButton>
+      </a>
+      <a :href="route('pirmakursiuStovyklos2022')">
+        <NButton round size="large" strong type="secondary"
```

Comment:
`type="secondary"` pasirinkimas neegzistuoja NButton komponente. Ir jeigu pažiūrėsi light mode, yra problema su šiuo mygtuku.

<img width="370" alt="image" src="https://github.com/vu-sa/vusa.lt/assets/23059846/f8c48636-dc0d-42de-ad13-2b6cd02d3354">


turėtų būti 

```suggestion
        <NButton round size="large" strong secondary
```

Reply:
resolved by suggestion